.github/actions/pr-comment/action.yml

```
@@ -5,6 +5,9 @@ inputs:
   name:
     description: 'Artifact name to import comment data from.'
     required: true
+  mode:
+    description: 'Mode of operation (upsert/recreate/delete).'
+    required: true
```

```suggestion
    default: 'upsert'
```

Would make the change in `bench-comment` obsolete. Just an idea.